docs/source/multidc/multidc.md

```diff
@@ -0,0 +1,455 @@
+```
+
+The expected output should look similar to the below:
+```console
+Datacenter: us-east-1
+=====================
+Status=Up/Down
+|/ State=Normal/Leaving/Joining/Moving
+--  Address      Load       Tokens       Owns    Host ID                               Rack
+UN  10.0.70.195  290 KB     256          ?       494277b9-121c-4af9-bd63-3d0a7b9305f7  c
+UN  10.0.59.24   559 KB     256          ?       a3a98e08-0dfd-4a25-a96a-c5ab2f47eb37  b
+UN  10.0.19.237  107 KB     256          ?       64b6292a-327f-4128-852a-6004039f402e  a
+```
+
+##### Retrieve PodIPs of ScyllaDB nodes for use as external seeds
+
+``` warning::
+    Due to the ephemeral nature of PodIPs, it is ill-advised to use them as seeds in production environments.
+    PodIPs are being used in this example for the sheer simplicity of this setup.
+    In production environments, it is recommended that you use domain names or non-ephemeral IP addresses.
+```
+
+Use the below commands and their expected outputs as a reference for retrieving the PodIPs used by the cluster for inter-node communication.
+```shell
+kubectl --context="${CONTEXT_DC1}" -n=scylla get pod/scylla-cluster-us-east-1-a-0 -o=go-template --template='{{ .status.podIP }}'
+```
+```console
+10.0.19.237
+```
+
+```shell
+kubectl --context="${CONTEXT_DC1}" -n=scylla get pod/scylla-cluster-us-east-1-b-0 -o=go-template --template='{{ .status.podIP }}'
+```
+```console
+10.0.59.24
+```
+
+```shell
+kubectl --context="${CONTEXT_DC1}" -n=scylla get pod/scylla-cluster-us-east-1-c-0 -o=go-template --template='{{ .status.podIP }}'
+```
+```console
+10.0.70.195
+```
+
+You are going to utilize the retrieved addresses as seeds for the other datacenter.
+
+#### Deploy the second datacenter
+
+To deploy the second datacenter, you will follow similar steps.
+
+First, create a dedicated 'scylla' namespace:
+```shell
+kubectl --context="${CONTEXT_DC2}" create ns scylla
+```
+
+Replace the values in `.spec.externalSeeds` of the below manifest with the Pod IP addresses that you retrieved earlier.
+The provided values are going to serve as initial contact points for the joining nodes of the second datacenter.
+
+For this guide, let's assume that the second cluster is running in `us-east-2` region and the nodes dedicated for running ScyllaDB nodes are running in zones `us-east-2a`, `us-east-2b` and `us-east-2c` correspondingly. If that is not the case, adjust the manifest accordingly.
+Having configured it, save the manifest as `dc2.yaml`:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+metadata:
+  name: scylla-cluster
+  namespace: scylla
+spec:
+  version: 5.2.7
+  agentVersion: 3.1.2
+  cpuset: true
+  exposeOptions:
+    broadcastOptions:
+      clients:
+        type: PodIP
+      nodes:
+        type: PodIP
+    nodeService:
+      type: Headless
+  externalSeeds:
+  - 10.0.19.237
+  - 10.0.59.24
+  - 10.0.70.195
+  datacenter:
+    name: us-east-2
+    racks:
+    - name: a
```

Comment:
specify `agentResources` to ensure Guaranteed QoS (in all examples)

Reply:
done